api/serializers.py

```diff
@@ -140,6 +140,12 @@ class Meta:
                   'request_reason', 'status', 'status_updater', 'helping_volunteer', 'created_date']
 
 
+class UpdateHelpRequestSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = HelpRequest
+        fields = ['notes', 'helping_volunteer', 'status', 'type_text']
```

Comment:
Need to add option to modify hamal representative who took care of task, like in the old system:
![image](https://user-images.githubusercontent.com/5032106/79672411-5a88f480-81da-11ea-89e7-6ad4340ce1a8.png)